libs/SalesforceReact/src/com/salesforce/androidsdk/reactnative/ui/SalesforceReactActivity.java

```diff
@@ -100,6 +103,9 @@ public void onDestroy() {
 
     @Override
     public void onResume() {
+        if(client != null && reactActivityDelegate != null){
+            reactActivityDelegate.onReadyCreate();
```

Comment:
In general, from a lifecycle perspective, I don't know if it's a good idea to call `onCreate()` from within `onResume()`. `onResume()` gets triggered every time this view loads, whereas `onCreate()` is meant for heavy lifting that's typically done once in the lifecycle of that view controller. If the parent activity does something different in `onCreate()` in a future version of `ReactNative`, this could be bad for us. Also, we lose the ability to load from saved `Bundle`. @wmathurin What do you think?